src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC } from "react";
+import { IPaginationProps } from "./pagination.interface";
+import { StylesWrapper } from "./pagination.styles";
+import {
+  BsChevronLeft,
+  BsChevronRight,
+  BsChevronDoubleLeft,
+  BsChevronDoubleRight,
+} from "react-icons/bs";
+
+export const Pagination: FC<IPaginationProps> = ({
+  totalPage,
+  currentPage,
+  onPageClick,
+}) => {
+  if (!totalPage || totalPage == 1) {
+    return null;
+  }
+
+  const getClassName = (id: number) => {
+    if (id + 1 === currentPage) return "page-button active";
+    return "page-button";
+  };
+
+  const getPagesValidity = (id: number) =>
```

Comment:
With utilities, it is recommended to separate them into a utils folder. If the utils are specific for this component, you can create a utils folder locally in the component folder. Else, if the function is meant to be used globally, you can expose it in a global utils folder :) This also provides further advantages, like, functions being scoped and pure, which allows you to create tests specifically to test the results of that function. :)

Reply:
🥔 noted, now this is moved to a utils folder specifically for the pagination component